### PR TITLE
Frontend/entity pages

### PR DESCRIPTION
**Pull request by batuhanovski**
View, Delete and Add Entity works properly.

For now, link entity is not implemented. (It is scheduled to next week.)

### CODE REVIEW

**Review by litityum — APPROVED** [review]
It is nice to see entities in action!

File: application/backend/main.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
+from flask import Flask, request
```
Comment (litityum) [inline application/backend/main.py:1]: This py file shouldn't be here

File: application/frontend/src/pages/AddEntity.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import * as React from "react";
+import {useParams} from "react-router";
+import {useEffect, useState} from "react";
+import axios from "axios";
+import {Link} from "react-router-dom";
+import {Button, Space, Table, Tag} from "antd";
+import { EntityForm } from "../components/EntityForm";
+
+const jwt = localStorage.getItem("jwt")
```
Comment (litityum) [inline application/frontend/src/pages/AddEntity.tsx:9]: jwt should change to token

File: application/frontend/src/pages/LinkEntity.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
+import * as React from "react";
+import {useParams} from "react-router";
+import {useEffect, useState} from "react";
+import axios from "axios";
+import {Link} from "react-router-dom";
+import {Button, Space, Table, Tag} from "antd";
+import { LinkEntityForm } from "../components/LinkEntityForm";
+
+const jwt = localStorage.getItem("jwt")
```
Comment (litityum) [inline application/frontend/src/pages/LinkEntity.tsx:9]: Also jwt here

**Review by litityum — CHANGES_REQUESTED** [review]
There are some changes needed

File: application/frontend/src/pages/AddEntity.tsx (hunk 2 above)
Comment (batuhanovski) [reply application/frontend/src/pages/AddEntity.tsx]: They are changed

**Pull request merged**